client/query.go

```diff
@@ -154,3 +155,48 @@ func Blocks(ctx context.CLIContext, startingHeight *big.Int) ([]store.Block, err
 
 	return blocks, nil
 }
+
+// Returns confirm sig results for given position
+// Trusts connected full node
+func Signatures(ctx context.CLIContext, position plasma.Position) ([]byte, error) {
+	key := store.GetOutputKey(position)
+	hash, err := ctx.QueryStore(key, store.DataStoreName)
+	if err != nil {
+		return nil, err
+	}
+
+	txKey := store.GetTxKey(hash)
+	txBytes, err := ctx.QueryStore(txKey, store.DataStoreName)
+
+	var tx store.Transaction
+	if err := rlp.DecodeBytes(txBytes, &tx); err != nil {
+		return nil, fmt.Errorf("transaction decoding failed: %s", err.Error())
+	}
+
+	// Look for confirmation signatures
+	// Ignore error if no confirm sig currently exists in store
+	var sigs []byte
+	if len(tx.SpenderTxs[position.OutputIndex]) > 0 {
+		queryPath := fmt.Sprintf("custom/%s/%s/%s",
+			store.QuerierRouteName, store.QueryTx, tx.SpenderTxs[position.OutputIndex])
+		data, err := ctx.Query(queryPath, nil)
+		if err != nil {
```

Comment:
Will an an err be returned if the confirm signatures are not present? I think `data == nil`.

If it doesn't return an error, should we return early here if `data == nil`

Reply:
This isn't done in any of our other usage of ctx.Query(). Do we want to add it in these other cases?